index.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+	<title>Javascript Quiz</title>
+</head>
+<body>
+	<form>
+		<label>height: <input type="text" id="height"/></label>
```

Comment:
The height will always need to be a number. Change the input type from text to number.